job-service-contract/src/main/resources/com/hpe/caf/services/job/swagger.yaml

```diff
@@ -335,6 +337,42 @@ definitions:
         example:
           "tag:4": "4"
           owner: "bob"
+  expiration-policy:
+    type: object
+    properties:
+      "Active":
+        $ref: "#/definitions/policy"
+      "Completed":
+        $ref: "#/definitions/policy"
+      "Failed":
+        $ref: "#/definitions/policy"
+      "Cancelled":
+        $ref: "#/definitions/policy"
+      "Waiting":
+        $ref: "#/definitions/policy"
+      "Paused":
+        $ref: "#/definitions/policy"
+    description: The expiration policy to be applied on the job
+  policy:
+    type: object
+    properties:
+      referenceDate:
+        type: string
+        enum:
+          - createDate
+          - lastUpdateDate
+        default: createDate
+      duration:
+        type: integer
+        format: int32
+        default: 5
```

Comment:
How about we combine `referenceDate` and `duration` into an `expiryTime`?  It could be an exact time like `2021-04-12T23:20:50.52Z`, or it could be an expression like `lastUpdateTime+P1D`.

Reply:
That would mean that they have to provide the date itself and not just its reference? I adapted the code to reflect that and now we have an expiryTime as a date-time (which is RFC 3339)